src/status_im2/common/data_store/wallet.cljs

```diff
@@ -1,11 +1,50 @@
 (ns status-im2.common.data-store.wallet
   (:require
-    clojure.set))
+    [clojure.set :as set]
+    [clojure.string :as string]
+    [status-im2.constants :as constants]))
+
+(defn chain-ids-string->set
+  [ids-string]
+  (->> (string/split ids-string constants/chain-id-separator)
+       (map js/parseInt)
+       (into #{})))
+
+(defn chain-ids-set->string
+  [ids]
+  (string/join constants/chain-id-separator ids))
+
+(defn rpc->account
+  [account]
+  (-> account
+      (set/rename-keys {:prodPreferredChainIds :prod-preferred-chain-ids
+                        :testPreferredChainIds :test-preferred-chain-ids
+                        :createdAt             :created-at
+                        :colorId               :color})
+      (update :prod-preferred-chain-ids chain-ids-string->set)
+      (update :test-preferred-chain-ids chain-ids-string->set)
+      (update :type keyword)
+      (update :color #(if (seq %) (keyword %) constants/account-default-customization-color))))
```

Comment:
We have a function to convert to kebab-case:
```clojure
(csk/->kebab-case :prodPreferredChainIds) 
;; => :prod-preferred-chain-ids
```

maybe we should update all the keys (`update-keys`) in this map using this function?

PLease check: https://clj-commons.org/camel-snake-kebab/

Reply:
Yes, that was my very first choice. But, recently encountered performance issues due to that https://github.com/status-im/status-mobile/issues/17282, which was solved with `rename-keys` fn. 

We only have four keys that need to be updated to the kebab case. I felt this plain old way was good enough as it's more performant.